bot/services/subscription_service.py

```
@@ -417,6 +420,119 @@ async def activate_trial_subscription(
             "subscription_url": final_subscription_url,
         }
 
+    async def _activate_traffic_package(
+        self,
+        session: AsyncSession,
+        user_id: int,
+        traffic_gb: float,
+        payment_amount: float,
```

Ruff (0.14.14)

[warning] Unused method argument: `payment_amount`

(ARG002)